src/build.py

```diff
@@ -623,16 +623,16 @@ def Apply(self, targets):
     if missing_names:
       raise Exception('Invalid step name(s): ' + str(missing_names))
 
-
     return [t for t in targets if self.Check(t.name)]
 
   def Check(self, target):
     """ Return true if the specified target will be run. """
     if self.include is not None:
       return target in self.include
 
-    return self.exclude is None or target not in self.exclude
-
```

Comment:
The flake8 didn't like the extra line here? Interesting I thought it liked two blank lines... I guess that's only for top-level definitions.